HOOKS.md

```diff
@@ -1410,6 +1411,45 @@ repos:
 You want to make sure that the field `depends_on` in each exposure is defined
 -----
 
+### `check-macro-is-referenced`
+
+ Ensures that the macro has been referenced in a model.
+
+ #### Arguments
+
+ `--manifest`: location of `manifest.json` file. Usually `target/manifest.json`. This file contains a full representation of dbt project. **Default: `target/manifest.json`**
+
+ #### Example
+
+ ```
+ repos:
+ - repo: https://github.com/offbi/pre-commit-dbt
+  rev: v0.1.1
+  hooks:
+  - id: check-macro-is-referenced
+ ```
+ #### When to use it
+
+ You want to make sure that all macros created are referenced at least in one of your models.
+
+ #### Requirements
+
+ | Macro exists in `manifest.json` <sup id="a1">[1](#f1)</sup> | Macro exists in `catalog.json` <sup id="a2">[2](#f2)</sup> |
```

Comment:
Not sure why we need HTML here

Reply:
Not sure either, but it's written the same way for other hooks...